src/IotHub/IotHub/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 * Add support to invoke failover for an IotHub to the geo-paired disaster recovery region.
+* Add support to manage message enrichment for an IotHub.
```

Comment:
I think it will be better if you add the name of added cmdlets in `ChangeLog`.

Reply:
Updated